doc/source/about.rst

```diff
@@ -1,5 +1,13 @@
 About libreant
 ================
 
-libreant is ubercool
+Libreant is a book manager for both digital and paper documents. It can store any kind of digital data actually, not only books.
+For it s db structure is highly customizable, documents can be archived on their types with differet metadata structure. The search function looks all over the db and rank matches with ElasticSearch 
+Elements into Libreant are defined as Items, for every item you can attach many files, usually this files are pdf or book scansions.
```

Comment:
they are actually "Volumes" not items, though I know that this new better terminology is not much used in the code, still.